test/controllers/property.controller.test.js

```diff
@@ -2429,18 +2427,20 @@ describe('Property Controller', () => {
       });
     });
 
-    context('without wrong vendor token', () => {
-      it('returns not found', (done) => {
-        request()
-          .get(`/api/v1/property/portfolio/${offer._id}`)
-          .set('authorization', newVendorToken)
-          .end((err, res) => {
-            expect(res).to.have.status(404);
-            expect(res.body.success).to.be.eql(false);
-            expect(res.body.message).to.be.eql('Portfolio not found');
-            done();
-          });
-      });
+    context('without wrong token without access', () => {
```

Comment:
Not sure of this your statement means, can you try to rewrite in a more readable format.

```suggestion
    context('with a different user', () => {
```